packages/neso-particles/package.py

```diff
@@ -33,4 +34,8 @@ def cmake_args(self):
         args = []
         if not "+build_tests" in self.spec:
             args.append("-DENABLE_NESO_PARTICLES_TESTS=off")
+        if "+nvcxx" in self.spec:
```

Comment:
Does this only work with hipsycl or can it be used with oneAPI as well? If it can't, you should add a [conflicts](https://spack.readthedocs.io/en/latest/packaging_guide.html#conflicts-and-requirements) statement.

Reply:
I don't think I can test this on the GPU node on CSD3 since it has an AMD CPU so I've only compiled with gcc. But I can add a conflicts statement and add an issue to the repo if anyone wants to test this with the oneAPI compiler.

Reply:
Added a conflict statement (see message for commit e2b6e79), also opened an issue if anyone wants to test the Intel OneAPI+Nvidia combination.